fontforgeexe/charinfo.c

```diff
@@ -3653,6 +3659,74 @@ static int CI_DefLCChange(GGadget *g, GEvent *e) {
 return( true );
 }
 
+// Turn the user's entered decomposition into the format returned by SFGetAlternate
+unichar_t* CI_ParseUserDecomposition(char* inp) {
+    unsigned long len = strlen(inp);
+    char* end;
+    unichar_t* out = malloc((len+1)*sizeof(unichar_t));
+
+    int j = 0;
+    for (unsigned long i = strtoul(inp, &end, 16); inp != end; i = strtoul(inp, &end, 16)) {
```

Comment:
So one thing I don't quite understand - there's references to this being limited to at most 5 characters, but this seems to allow for an arbitrary length?

Reply:
I didn't think this was a big problem, but it is inconsistent, so I limited it.

Reply:
One area where it might have been a problem is `codepoints_as_hex`, since there's an assumption of a max of 5